astrokat/test/test_nd_timings.py

```diff
@@ -19,27 +19,44 @@
 # - nd-pattern-plus-off
 # - nd-trigger
 
-class test_astrokat_yaml(unittest.TestCase):
+PROPOSAL_ID = "CAM_AstroKAT_UnitTest"
+OBSERVER = "KAT Tester"
 
+TESTS_PATH = os.path.abspath(os.path.dirname(__file__))
+
+
+class test_astrokat_yaml(unittest.TestCase):
```

Comment:
Can we fix these class names, while we are here?  CamelCaps, please.

Reply:
Will do.

Reply:
Still need to fix this.

Reply:
Still need to fix this - why is conversation set to resolved again?